Where to go for help (#264)
agbrs/agb: README.md

There is an (in progress) tutorial which you can find on the [project website](https://agbrs.github.io/agb/).

## Help / Support

If you need any help, the [discussions page](https://github.com/agbrs/agb/discussions)
is a great place to get help from the creators and contributors.

> Here you can create a new discussion with the Q&A category and help should be on the way.
> 

Reviewer: How about:

```suggestion
Feel free to create a new discussion in the Q&A category and we'll do our best to help!

```
